### PR TITLE
adding a 3rd parameter to saveAs to allow other mimeTypes, such as im…

### PR DESCRIPTION
**Pull request by leonardorame**
…age/jpeg.

### CODE REVIEW

**Comment by leonardorame** [comment]
Hi Erik, could you review this again?.

File: src/imageTools/saveAs.js [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { $ } from '../externalModules.js';
 
-export default function (element, filename) {
+export default function (element, filename, mimetype) {
```
Comment (swederik) [inline src/imageTools/saveAs.js]: Can you make "image/png" the default mimetype?

**Comment by swederik** [comment]
Thanks!

File: src/imageTools/saveAs.js (hunk 1 above)
Comment (leonardorame) [reply src/imageTools/saveAs.js]: I made it the *old way*:

    // setting the default value for mimetype to image/png
    mimetype = mimetype || "image/png";

Of course I can change to `param = defaultvalue`, do you prefer that way?.

Comment (swederik) [reply src/imageTools/saveAs.js]: It's a minor difference, I guess there's no need to change it. I realized it didn't matter and just merged it :-) Thanks!

**Pull request merged**